lib/components/RunFrame/RunFrame.tsx

```diff
@@ -258,6 +258,17 @@ export const RunFrame = (props: RunFrameProps) => {
       props.onRenderFinished?.({ circuitJson })
       setAutoroutingGraphics({})
 
+      console.log(
+        circuitJson.filter(
+          (x: any) => x.type.includes("error") && x.error_type,
+        ),
+      )
+      circuitJson
+        .filter((x: any) => x.type.includes("error") && x.error_type)
+        .forEach((x) =>
+          setError({ error: (x as any).message || "Unknown error" }),
```

Comment:
We should support multiple errors in some way, this may need a design reference (eg nextjs and vite have a way of displaying multiple errors)